src/SFA.DAS.EAS.Account.Worker.UnitTests/Foo.cs

```diff
@@ -0,0 +1,13 @@
+﻿using NUnit.Framework;
```

Comment:
Is this needed? Looks like its a placeholder that never got replaced.

Reply:
Yeah, if there's no tests in the assembly the build fails so I added a placeholder. I'd delete the project but it's being re-purposed by Ian on his branch.